c:\users\fadi\appdata\local\programs\python\python36-32